spec/factories/user_factory.rb

```diff
@@ -0,0 +1,15 @@
+FactoryGirl.define do
+
+  factory :user do
+    name                  'Person'
+    username              'pers0n'
+    email                 'person@mail.com'
+    password              'bananas!'
+    password_confirmation 'bananas!'
+  end
+
+  factory :confirmed_user, :parent => :user do
+    after(:create) { |user| user.confirm! }
```

Comment:
Style/SymbolProc: Pass `&:confirm!` as an argument to `after` instead of a block.